### PR TITLE
How to upgrade minikube

### PR DESCRIPTION
**Pull request by RA489**
How to upgrade minikube
fixes https://github.com/kubernetes/website/issues/15950

### CODE REVIEW

**Comment by RA489** [comment]
Ref https://github.com/kubernetes/website/issues/16866

File: content/en/docs/setup/learning-environment/minikube.md [hunk 1]
```diff
@@ -329,6 +329,9 @@ Starting the cluster again will restore it to its previous state.
 The `minikube delete` command can be used to delete your cluster.
 This command shuts down and deletes the Minikube Virtual Machine. No data or state is preserved.
 
+### Upgrading minikube
+See [upgrade minikube](https://minikube.sigs.k8s.io/docs/start/macos/)
```
Comment (sftim) [inline content/en/docs/setup/learning-environment/minikube.md]: Is that the right hyperlink?

Comment (NatsumiHB) [reply content/en/docs/setup/learning-environment/minikube.md:333]: I know I'm late by a year and don't know if that got fixed but the link shouldn't include the `/macos/` at the end.

Comment (sftim) [reply content/en/docs/setup/learning-environment/minikube.md:333]: https://k8s.io/docs/setup/learning-environment/minikube/ is now a redirect, as it goes.

**Pull request merged**